web/src/components/Ticket/Ticket.js

```diff
@@ -51,15 +58,11 @@ const Ticket = ({ ticket }) => {
       <div className="rw-segment">
         <header className="rw-segment-header">
           <h2 className="rw-heading rw-heading-secondary">
-            Ticket {ticket.id} Detail
+            Ticket {ticket.number} Detail
```

Comment:
I am playing around with what to show here.  I think we may want to drop ticket.number and use ticket.id instead.  No reason to track two numbers for the same record..  This update is just changing what we see on the ticket form.